frontend/test/pytest/test_debug.py

```diff
@@ -447,37 +447,35 @@ def f(x: float):
             ),
             (
                 "HLOLoweringPass",
-                "%2 = arith.mulf %in, %in_0 : f64\n",
-                "%t = arith.mulf %in, %in_0 : f64\n" + "    %2 = arith.mulf %t, %in_0 : f64\n",
+                "%0 = arith.mulf %extracted, %extracted : f64\n",
```

Comment:
@tzunghanjuang I had to fix this test following the introduction of detensorize. Can you confirm that is still OK?